lib/bot.js

```diff
@@ -430,7 +430,7 @@ module.exports = (function() {
 			});
 		},
 
-		edit: function(title, content, summary, callback) {
+		edit: function(title, content, summary, minor, callback) {
```

Comment:
I carefully read the given lines of getUser() and the [example](https://github.com/ValentinBrclz/nodemw/blob/devel/examples/getUsers.js), but I am not sure how you see this... Could you be more specific on how you would implement it?